src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ elseif(CLR_CMAKE_PLATFORM_ARCH_ARM64)
     return()
 endif()
 
+if(CLR_CMAKE_PLATFORM_LINUX_ALPINE)
+    message(WARNING "lldb is not supported on Alpine Linux yet")
```

Comment:
I don't understand conditions against few of few hundreds of Linux distributions...